test/api-documentation.test.js

```diff
@@ -571,4 +583,247 @@ describe('<api-documentation>', function() {
       assert.notOk(node, 'method is not endered');
     });
   });
+
+  [
+    ['Compact model', true],
+    ['Regular model', false]
+  ].forEach(([name, compact]) => {
+    describe(name, () => {
+      async function narrowFixture({ amf, selectedType, renderSlots = true }){
+        return await fixture(html`
+          <api-documentation .amf="${amf}" .selectedType="${selectedType}" narrow>
+            ${renderSlots
```

Comment:
This time I won't be asking you to change this but in the future try to create two fixtures. One that renders the element with the children and the other without them. The reason for that is that it's easier to understand the code and realize what is happening. The next person updating the component will spend less time to analyze it.